### ISSUE TITLE
CVE-2020-7677 - eval in thenify 3.3.0

### ISSUE THREAD

**Issue by denodaeus** [lerna/lerna#2681]
https://snyk.io/vuln/SNYK-JS-THENIFY-571690

```
Affected versions of this package are vulnerable to Arbitrary Code Execution. The name argument provided to the package can be controlled by users without any sanitization, and this is provided to the eval function without any sanitization.
```

## Expected Behavior
No vulnerability

## Current Behavior
Raises a high vulnerability on security scans for various services (such as synk.io)

## Possible Solution
Update `thenify` to >= 3.3.1

**Comment by denodaeus** [comment]
This can be mitigated by updating @lerna/create-symlinks dependency on #zkochan/cmd-shim to the latest (currently 3.1.0 from more than a couple of years ago, latest available is 5.0.0):

https://snyk.io/test/npm/@zkochan/cmd-shim/3.1.0?tab=dependencies

**Comment by JamesHenry** [comment]
Following on from the note above, in looking at this one specifically, it is no longer applicable. The packages/versions mentioned are no longer an issue.

Many thanks @denodaeus!
